client/src/features/sessionsV2/ShowSessionPage.tsx

```diff
@@ -362,3 +380,153 @@ function DeleteSessionBtn({ openDeleteSession }: DeleteSessionBtnProps) {
+  session?: Session;
+  namespace?: string;
+  slug?: string;
+}) {
+  const [isOpen, setIsOpen] = useState(false);
+  const { projectId, launcherId } = useMemo(() => {
+    if (session == null) {
+      return { projectId: undefined, launcherId: undefined };
+    }
+    const annotations = NotebooksHelper.cleanAnnotations(
+      session.annotations
+    ) as NotebookAnnotations;
+    return {
+      projectId: annotations.projectId,
+      launcherId: annotations.launcherId,
+    };
+  }, [session]);
+
+  const {
+    data: launchers,
+    isLoading: isLoadingLaunchers,
+    error: launchersError,
+  } = useGetProjectSessionLaunchersQuery({ projectId: projectId ?? "" });
+  const { data: project, isLoading: isLoadingProject } =
+    useGetProjectsByNamespaceAndSlugQuery(
+      namespace && slug ? { namespace, slug } : skipToken
+    );
+
+  const launcher = useMemo(
+    () => launchers?.find(({ id }) => id === launcherId),
+    [launcherId, launchers]
+  );
+  const toggle = useCallback(() => {
+    setIsOpen((open) => !open);
+  }, []);
+
+  const projectUrl =
+    project &&
+    generatePath(ABSOLUTE_ROUTES.v2.projects.show.root, {
+      namespace: project?.namespace,
+      slug: project?.slug,
+    });
+
+  if (isLoadingLaunchers || isLoadingProject) {
+    return (
+      <div className={cx("d-flex", "align-items-center")}>
+        <p className={cx("text-white", "mb-0")}>
+          <Loader inline size={16} /> Checking session details...
+        </p>
+      </div>
+    );
+  }
+  if (launchersError || !launcher)
+    return (
+      <div className={cx("d-flex", "align-items-center")}>
+        <p className={cx("text-white", "mb-0")}>
+          <ExclamationTriangle className="bi" /> Session not accessible
+        </p>
+      </div>
+    );
+  const detailsModal = project && session && projectUrl && (
+    <Modal backdrop="static" centered isOpen={isOpen} size="lg" toggle={toggle}>
+      <ModalHeader toggle={toggle}>Session details {launcher.name}</ModalHeader>
+      <ModalBody>
+        <div className={cx("d-flex", "flex-column", "gap-3")}>
+          <div>
+            <p className="mb-0">
+              <Briefcase className={cx("bi", "me-2")} />
+              Project:{" "}
+              <Link to={projectUrl}>
+                <span className="fw-bold">{project.name}</span>
+              </Link>
+            </p>
+          </div>
+          <div>
+            <p className="mb-0">
+              <Clock className={cx("bi", "me-2")} />
+              <span className="fw-bold">
+                <TimeCaption
+                  prefix="Launched"
+                  datetime={session.started}
+                  className={cx("fs-6")}
+                />
+              </span>
+            </p>
+          </div>
+          <div
+            className={cx(
+              "d-block",
+              "d-lg-flex",
+              "pb-2",
+              "pb-lg-0",
+              "gap-2",
+              "align-items-center"
+            )}
```

Comment:
No need to add additional space on the bottom since the spacing is already handled by the parent component with `gap-3`

```suggestion
            className={cx(
              "d-block",
              "d-lg-flex",
              "gap-2",
              "align-items-center"
            )}
```

Reply:
thanks @lorenzo-cavazzi I have removed the unnecessary style